WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -146,6 +146,15 @@ private void navigateReader() {
                         childAtPosition(withId(R.id.bottom_navigation), 0), 1)));
         waitForElementUntilDisplayed(navReaderButton).perform(click());
 
+        // Select Discover
+        ViewInteraction spinner = onView(
+                allOf(withId(R.id.filter_spinner), childAtPosition(
+                        withId(R.id.toolbar_with_spinner),0)));
+        waitForElementUntilDisplayed(spinner).perform(click());
+
+        ViewInteraction spinnerOption = onView(allOf(withId(R.id.text), withText("Discover")));
```

Comment:
I don't like hardcoding the string here but since this is dynamically generated, I'm not aware of another way to do this right now. If it becomes a problem as we expand to other locales, I will revisit.